{{cookiecutter.project_slug}}/.github/workflows/workflow.yml

```diff
@@ -35,4 +35,4 @@ jobs:
       - name: Tests
         run: npm run test:ci
       - name: Codecov test coverage
-        run: bash scripts/coverage.sh "${{ matrix.os }}" "${{ matrix.node-version }}"
+        run: npm run test:report:coverage "${{ matrix.os }}" "${{ matrix.node-version }}"
```

Comment:
Good call :+1:

Reply:
Going to update this as an additional step is required for Windows machines:
https://github.com/netlify/cli/runs/1373808824?check_suite_focus=true#step:8:8
https://github.com/netlify/cli/pull/1532/commits/efc2143a78579998168244e28c0029d72d12a3ba#diff-7829468e86c1cc5d5133195b5cb48e1ff6c75e3e9203777f6b2e379d9e4882b3R40

Reply:
This should be fixed by `eslint-config-node@0.6.1` (see https://github.com/netlify/cli/pull/1542/files)

Reply:
Resolved by https://github.com/netlify/node-template/pull/4/commits/ff887c8121979d2c995ffdfdb2ebe0337e78581d

Reply:
Could we use the new test coverage setup?
See https://github.com/netlify/cli/pull/1542

Reply:
Sure, going to update this soon

Reply:
Done in https://github.com/netlify/node-template/pull/4/commits/6f25f9f9f2c0ae8ce4fc74e0824ff52fdcc5ac8c